interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Selection and Navigation
+author: Alexey Kazakov
+legacyId: 4533
+---
+# Selection and Navigation
+## Navigation
+Use the Tree List's pager to switch between pages.
```

Comment:
pager?

Reply:
Pager - is a panel at the bottom of the treelist that enables page navigation.